pkg/output/progress_test.go

```diff
@@ -7,13 +7,20 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestProgressTimeout(t *testing.T) {
+func TestProgressTimeoutDoesNotInc(t *testing.T) {
+	progress := NewProgress()
+	progress.Start()
+	progress.Stop() // should not hang
+	progress.Inc()  // should not hang
+	assert.Equal(t, progress.Val(), uint64(0))
```

Comment:
You mixed expected and actual value here

```diff
-assert.Equal(t, progress.Val(), uint64(0))
+assert.Equal(t, uint64(0), progress.Val())
```

Reply:
:+1: fixed